GeoHealthCheck/config_main.py

```diff
@@ -148,5 +148,8 @@
     },
     'FTP': {
         'probe_class': None
-    }
+    },
+    'Custom:GeoNode': {
```

Comment:
Perhaps `OSGeo:GeoNode` instead? We should update/add to https://github.com/OSGeo/Cat-Interop/blob/master/LinkPropertyLookupTable.csv at the same time.

Reply:
yep, that would be better.